c7n/resources/awslambda.py

```diff
@@ -60,6 +60,7 @@ class AWSLambda(query.QueryResourceManager):
 
     class resource_type(query.TypeInfo):
         service = 'lambda'
+        arn = 'FunctionArn'
```

Comment:
sans this arn was constructed, but typically left off the version/alias identifier.